yoti_python_sdk/__init__.py

```diff
@@ -23,19 +24,20 @@
 
 __version__ = main_ns["__version__"]
 YOTI_API_URL = environ.get("YOTI_API_URL", DEFAULTS["YOTI_API_URL"])
+DOCS_API_URL = environ.get("DOCS_API_URL", DEFAULTS["DOCS_API_URL"])
```

Comment:
```suggestion
YOTI_DOC_SCAN_API_URL = environ.get("YOTI_DOC_SCAN_API_URL", DEFAULTS["YOTI_DOC_SCAN_API_URL"])
```

Reply:
Might only need `YOTI_` prefix on the environment variable:
```suggestion
DOC_SCAN_API_URL = environ.get("YOTI_DOC_SCAN_API_URL", DEFAULTS["DOC_SCAN_API_URL"])
```

Reply:
This has been addressed in 9077b7a.  Let me know what you think